include/shark/ObjectiveFunctions/Benchmarks/MultiObjectiveBenchmark.h

```diff
@@ -0,0 +1,189 @@
+//===========================================================================
+/*!
+ * 
+ *
+ * \author     Oswin Krause
+ * \date        2016
+ *
+ *
+ * \par Copyright 1995-2015 Shark Development Team
+ * 
+ * <BR><HR>
+ * This file is part of Shark.
+ * <http://image.diku.dk/shark/>
+ * 
+ * Shark is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published 
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ * 
+ * Shark is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ * 
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Shark.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+//===========================================================================
+#ifndef SHARK_OBJECTIVEFUNCTIONS_BENCHMARK_MULTIOBJECTIVEBENCHMARK_H
+#define SHARK_OBJECTIVEFUNCTIONS_BENCHMARK_MULTIOBJECTIVEBENCHMARK_H
+
+#include <shark/ObjectiveFunctions/AbstractObjectiveFunction.h>
+#include <shark/ObjectiveFunctions/Benchmarks/RotatedErrorFunction.h>
+#include <shark/ObjectiveFunctions/BoxConstraintHandler.h>
+#include <shark/Rng/GlobalRng.h>
+
+#include <shark/LinAlg/rotations.h>
+#include <tuple>
+
+namespace shark {
+
+namespace detail{
+	//taken from the web. implements an std::integer_sequence type representing a sequence 0,...,N-1, std::integer_sequence is not here until C++14
```

Comment:
stackoverflow. so this is safe, probably also the standard implementation since c++14